src/main/java/blusunrize/immersiveengineering/common/blocks/metal/conveyors/DropConveyor.java

```diff
@@ -34,6 +36,9 @@
 public class DropConveyor extends BasicConveyor
 {
 	public static final ResourceLocation NAME = new ResourceLocation(MODID, "dropper");
+	private static final VoxelShape REQ_SPACE = VoxelShapes.create(0.25, 0.75, 0.25, 0.75, 1.0, 0.75);
```

Comment:
I would prefer a slightly longer name here (probably `REQUIRED_SPACE`), but it's not that important.